src/agent/src/main.rs

```diff
@@ -478,13 +482,23 @@ async fn launch_guest_component_procs(logger: &Logger, config: &AgentConfig) ->
     }
 
     debug!(logger, "spawning attestation-agent process {}", AA_PATH);
+    let mut aa_args = vec!["--attestation_sock", AA_ATTESTATION_URI];
+    let initdata_parameter;
+    if let Some(initdata_hash) = INITDATA.get() {
```

Comment:
instead of using a global variable, can we return `INITDATA` (the b64 encoded digest) from `initialize_initdata()`?